unittest/test_controls.cpp

```diff
@@ -0,0 +1,101 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2021, LAAS-CNRS, New York University, Max Planck Gesellschaft,
+//                     University of Edinburgh, INRIA, University of Trento
+// Copyright note valid unless otherwise controld in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#define BOOST_TEST_NO_MAIN
+#define BOOST_TEST_ALTERNATIVE_INIT_API
+
+#include "factory/control.hpp"
+#include "crocoddyl/core/numdiff/control.hpp"
+#include "unittest_common.hpp"
+
+using namespace boost::unit_test;
+using namespace crocoddyl::unittest;
+
+//----------------------------------------------------------------------------//
+
+void test_Dvalue_num_diff(ControlTypes::Type control_type) {
```

Comment:
Rename the `test_` functions with the agreed names discussed in the Control API.